docs/customerContact.rst

```diff
@@ -0,0 +1,71 @@
+.. _customerContact:
+
+Customer Contact Information Reference
+======================================
+
+When a customer enables your Alexa skill, your skill can request the customer's permission to the their contact information, which includes name, email address and phone number, if the customer has consented. You can then use this data to support personalized intents to enhance the customer experience without account linking. For example, your skill may use customer contact information to make a reservation at a nearby restaurant and send a confirmation to the customer.
+
+.. js:function:: CustomerContact.constructor(alexaEvent)
+
+  Constructor
+
+  :param alexaEvent: Alexa Event object.
+
+.. js:function:: CustomerContact.getAddress()
+
+  Gets full address info
+
+  :returns Object: A JSON object with the full address info
+
+.. js:function:: CustomerContact.getCountryRegionPostalCode()
+
+  Gets country/region and postal code
+
+  :returns Object: A JSON object with country/region info
+
+With Voxa, you can ask for the user's full name like this:
+
+.. code-block:: javascript
+
+  const skill = new Voxa({ variables, views });
+
+  skill.onIntent('FullAddressIntent', async (alexaEvent) => {
+    const name = await alexaEvent.customerContact.getName();
+
+    alexaEvent.model.name = name;
+    return { reply: 'CustomerContact.Name' };
+  });
+
+Voxa also has a method to request all parameters at once:
+
+.. code-block:: javascript
+
+  const skill = new Voxa({ variables, views });
+
+  skill.onIntent('FullAddressIntent', async (alexaEvent) => {
+    const info = await alexaEvent.customerContact.getFullUserInformation();
```

Comment:
seems like `getFullUserInformation` is missing from the API described in this documentation

Reply:
True that